content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -0,0 +1,337 @@
+    http:
+    - match:
+      - port: 15012
+      route:
+      - destination:
+          host: istiod.external-istiod.svc.cluster.local
+          port:
+            number: 15012
+    - match:
+      - port: 15017
+      route:
+      - destination:
+          host: istiod.external-istiod.svc.cluster.local
+          port:
+            number: 443
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: external-istiod-dr
+  namespace: external-istiod
+spec:
+  host: istiod.external-istiod.svc.cluster.local
+  trafficPolicy:
+    portLevelSettings:
+    - port:
+        number: 15012
+      tls:
+        mode: SIMPLE
+      connectionPool:
+        http:
+          h2UpgradePolicy: UPGRADE
+    - port:
+        number: 443
+      tls:
+        mode: SIMPLE
+EOF
+{{< /text >}}
+
+Apply the Istio configuration in `external_cluster`:
+
+{{< text bash >}}
+$ kubectl apply -f external-istiod-gw.yaml --context="${CTX_EXTERNAL_CLUSTER}"
+{{< /text >}}
+
+### Setup remote cluster
+
+Generate the Istio configuration for `remote_cluster` and the `external-istiod` namespace:
+
+{{< text bash >}}
+$ cat <<EOF > remote-config-cluster.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+ namespace: external-istiod
+spec:
+  profile: remote
+  meshConfig:
+    rootNamespace: external-istiod
+    defaultConfig:
+      discoveryAddress: $REMOTE_ISTIOD_ADDR:15012
+      proxyMetadata:
+        XDS_ROOT_CA: /etc/ssl/certs/ca-certificates.crt
+        CA_ROOT_CA: /etc/ssl/certs/ca-certificates.crt
+  components:
+    pilot:
+      enabled: false
+    istiodRemote:
+      enabled: true
+
+  values:
+    global:
+      caAddress: $REMOTE_ISTIOD_ADDR:15012
+      istioNamespace: external-istiod
+      meshID: mesh1
+      multiCluster:
+        clusterName: remote_cluster
+      network: network1
+    istiodRemote:
+      injectionURL: https://$REMOTE_ISTIOD_ADDR:15017/inject
+    base:
+      validationURL: https://REMOTE_ISTIOD_ADDR:15017/validate
+EOF
+{{< /text >}}
+
+Install the configuration in `remote_cluster`:
+
+{{< text bash >}}
+$ istioctl apply -f remote-config-cluster.yaml --context="${CTX_REMOTE_CLUSTER}"
+{{< /text >}}
+
+You may notice the ingress gateway in `remote_cluster` is not running yet.
+This is expected until the external istiod is running, which you will install next.
+
+### Setup external istiod in the control plane cluster
+
+Create remote secret to allow external istiod in `external_cluster` to
+access the `remote_cluster`:
+
+{{< text bash >}}
+$ kubectl create sa istiod-service-account -n external-istiod --context="${CTX_EXTERNAL_CLUSTER}"
+$ istioctl x create-remote-secret \
+  --context="${CTX_REMOTE_CLUSTER}" \
+  --type=config \
+  --namespace=external-istiod | \
+  kubectl apply -f - --context="${CTX_EXTERNAL_CLUSTER}"
+{{< /text >}}
+
+Generate the Istio configuration for the `external-istiod`
+namespace in `external_cluster`:
+
+{{< text bash >}}
```

Comment:
Also need to specify cluster name and network here

Reply:
Updated with cluster name and mesh id.  left network out per our discussion